Notely/lib/screens/authenticate/sign_in.dart

```diff
@@ -0,0 +1,90 @@
+import 'package:Notely/services/auth.dart';
+import 'package:flutter/material.dart';
+
+class SignIn extends StatefulWidget {
+  final Function toggleView;
+
+  SignIn({this.toggleView});
+
+  @override
+  _SignInState createState() => _SignInState();
+}
+
+class _SignInState extends State<SignIn> {
+  final AuthService _auth = AuthService();
+  final _formKey = GlobalKey<FormState>();
+
+  String email = '';
+  String password = '';
+  String error = '';
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        backgroundColor: Colors.white,
+        appBar: AppBar(
+          backgroundColor: Colors.tealAccent.shade400,
```

Comment:
if you want to change the color, you have to go in and change it in multiple files. This is not desirable.